core/utils/dom.js

```diff
@@ -57,36 +58,35 @@ Blockly.utils.dom.NodeType = {
  * @type {Object}
  * @private
  */
-Blockly.utils.dom.cacheWidths_ = null;
+let cacheWidths = null;
 
 /**
  * Number of current references to cache.
  * @type {number}
  * @private
  */
-Blockly.utils.dom.cacheReference_ = 0;
+let cacheReference = 0;
 
 /**
  * A HTML canvas context used for computing text width.
  * @type {CanvasRenderingContext2D}
  * @private
  */
-Blockly.utils.dom.canvasContext_ = null;
+let canvasContext = null;
 
 /**
  * Helper method for creating SVG elements.
- * @param {string|Blockly.utils.Svg<T>} name Element's tag name.
+ * @param {string|Svg<T>} name Element's tag name.
  * @param {!Object} attrs Dictionary of attribute names and values.
  * @param {Element=} opt_parent Optional parent on which to append the element.
  * @return {T} Newly created SVG element.  The return type is {!SVGElement} if
- *     name is a string or a more specific type if it a member of
- *     Blockly.utils.Svg
+ *     name is a string or a more specific type if it a member of Svg.
  * @template T
  */
-Blockly.utils.dom.createSvgElement = function(name, attrs, opt_parent) {
-  var e = /** @type {T} */
-      (document.createElementNS(Blockly.utils.dom.SVG_NS, String(name)));
-  for (var key in attrs) {
+const createSvgElement = function(name, attrs, opt_parent) {
+  const e = /** @type {T} */
+      (document.createElementNS(SVG_NS, String(name)));
+  for (const key in attrs) {
```

Comment:
No, because it is redefined every iteration; it's not the same variable taking on a new value, it's a new variable entirely each time through the loop.